tests/core/mempool/test_mempool_manager.py

```diff
@@ -534,3 +535,93 @@ async def test_get_items_not_in_filter() -> None:
     sb2_and_3_filter = PyBIP158([bytearray(sb2_name), bytearray(sb3_name)])
     result = mempool_manager.get_items_not_in_filter(sb2_and_3_filter)
     assert result == [sb1]
+
+
+def always(_: bytes32) -> bool:
+    return True
+
+
+def never(_: bytes32) -> bool:
+    return False
+
+
+class TestFor(Enum):
+    MAX_COST = 0
+    MAX_FEE = 1
+    NORMAL_PATH = 2
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("test_for", [t for t in TestFor])
```

Comment:
these appear to be 3 independent tests. not really ideal to parameterize like this. It would be different if they would run through the same code and test some expected outcome (that can be specified as a parameter, and vary across the test cases).